examples/line.py

```diff
@@ -0,0 +1,45 @@
+import numpy as np
+import vedo
+
+vedo.settings.default_backend = "vtk"
+
+from brainrender import Scene
+from brainrender.actors import Points, Line
+
+# Display the Allen Brain mouse atlas at 25 micrometer resolution.
```

Comment:
brainrender works in real world units, so I suggest removing the reference to resolution so that users don't think the resolution is relevant. 
```suggestion
# Display the Allen Brain mouse atlas
```

Reply:
Addressed in 44d7ab2.